shared/chat/conversation/info-panel/add-people.js

```diff
@@ -97,4 +100,8 @@ const AddPeople = compose(
   OverlayParentHOC
 )(_AddPeople)
 
+const styles = Styles.styleSheetCreate({
+  addButton: {marginLeft: Styles.globalMargins.small, marginRight: Styles.globalMargins.small},
```

Comment:
This should be padding on the container I think. On mobile the button overflows

<img width="337" alt="image" src="https://user-images.githubusercontent.com/11968340/56814305-e5742080-680c-11e9-9ce4-49ee2c0ce9da.png">